setup.py

```diff
@@ -35,14 +36,16 @@ def read(fname):
     url='https://github.com/gojuno/make-profiler',
     packages=['make_profiler'],
     test_suite='test',
+    zip_safe=False,
     install_requires=(
         'more-itertools==2.4.1',
     ),
     entry_points={
         'console_scripts': [
             'profile_make_clean = make_profiler.cmd_clean:main',
             'profile_make = make_profiler.__main__:main',
-            'profile_make_lint = make_profiler.lint_makefile:main'
+            'profile_make_lint = make_profiler.lint_makefile:main',
+            'profile_make_viewer = make_profiler.viewer_export:main'
```

Comment:
since these copied once, then how about something like this?
```suggestion
            'profile_make_init_viewer = make_profiler.viewer_export:main'
```